app/assets/stylesheets/annotations.scss

```diff
@@ -937,5 +937,9 @@ span > .material-icons {
   font-size: 14px !important;
   box-shadow: 0.5px 0.5px 0.5px 0.5px grey !important;
   border-color: white !important;
-} 
+}
+
+.hljs-comment span {
+  color: #008000 !important;
```

Comment:
Can this be a variable? see _variables.scss

Reply:
Okay, changed this.